samples/v1.0/Elements/Image.Explicit.Size.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Can we add a person type example as well to exercise the ellipse code path?

Reply:
sure

---
In reply to: [184109786](https://github.com/Microsoft/AdaptiveCards/pull/1373#discussion_r184109786) [](ancestors = 184109786)